src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -56,12 +57,17 @@ public OverseasEntitySubmissionDtoValidator(EntityDtoValidator entityDtoValidato
     }
 
     public Errors validate(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
-        entityDtoValidator.validate(overseasEntitySubmissionDto.getEntity(), errors, loggingContext);
-        presenterDtoValidator.validate(overseasEntitySubmissionDto.getPresenter(), errors, loggingContext);
+        if (UtilsValidators.isNotNull(overseasEntitySubmissionDto.getEntity(), OverseasEntitySubmissionDto.ENTITY_FIELD, errors, loggingContext)) {
+            entityDtoValidator.validate(overseasEntitySubmissionDto.getEntity(), errors, loggingContext);
+        }
+
+        if (UtilsValidators.isNotNull(overseasEntitySubmissionDto.getPresenter(), OverseasEntitySubmissionDto.PRESENTER_FIELD, errors, loggingContext)) {
+            presenterDtoValidator.validate(overseasEntitySubmissionDto.getPresenter(), errors, loggingContext);
+        }
 
         var dueDiligenceDto = overseasEntitySubmissionDto.getDueDiligence();
         var overseasEntityDueDiligenceDto = overseasEntitySubmissionDto.getOverseasEntityDueDiligence();
-        if (dueDiligenceDataBlockValidator.onlyOneBlockPresent(dueDiligenceDto, overseasEntityDueDiligenceDto, errors, loggingContext)) {
+        if (dueDiligenceDataBlockValidator.onlyOneCorrectBlockPresent(overseasEntitySubmissionDto.getWhoIsRegistering(), dueDiligenceDto, overseasEntityDueDiligenceDto, errors, loggingContext)) {
```

Comment:
i don't think we need the who_is_registering as the previous check would have been sufficient. It would check that only one of either due diligence or OE due diligence were present. So if only one was present we can assume that is the correct one.